intercom-java/src/main/java/io/intercom/api/Event.java

```diff
@@ -25,6 +35,32 @@ public static void create(Event event) throws InvalidException, AuthorizationExc
         DataResource.create(event, "events", Void.class);
     }
 
+    public static Job submit(List<JobItem<Event>> items)
+        throws AuthorizationException, ClientException, ServerException, InvalidException, RateLimitException {
+        return submit(items, null);
+    }
+
+    public static Job submit(List<JobItem<Event>> items, Job job)
+        throws AuthorizationException, ClientException, ServerException, InvalidException, RateLimitException {
+        return Job.submit(validateJobItems(items), job, BULK_PATHS);
+    }
+
+    public static JobItemCollection<Event> listJobErrorFeed(String jobID)
+        throws AuthorizationException, ClientException, ServerException, InvalidException, RateLimitException {
+        return Job.listJobErrorFeed(jobID, Event.class);
```

Comment:
The underlying Job doesn't care about specific types too much (it's looking for a TypedData), but the helpers on Event/User force you to say what things are.